app/javascript/utils/DelegationRequests.js

```diff
@@ -39,4 +40,18 @@ export async function delegateQuestions(answers, buildingId, email, firstName, l
     });
     addAnswers(answersToUpdate, buildingId);
   } catch (error) {}
+}
+
+export async function delegateBuildingQuestions(buildingId, userDetails, state, addAnswers, callbackFn=null) {
+  let answers = getAnswersForBuilding(buildingId, state);
+
+  await delegateQuestions(answers, buildingId, userDetails.email, userDetails.firstName, userDetails.lastName, addAnswers);
+  callbackFn ? callbackFn() : null;
+}
+
+export async function delegateCategoryQuestions(categoryId, buildingId, userDetails, state, addAnswers, callbackFn=null) {
```

Comment:
Just wondering, what's the purpose of having a callback function here? When it's used it isn't passed a callback function anyways; when would we need to pass it a callback?

Reply:
We were originally planning on using the callback to tell the confirmation modal to close, but it turns out we can just have the modal `await` the request (Ethan's branch) so we can remove this now